server/migrations/utils/dumpTable.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('child_process');
+const path = require('path');
+
+/**
+ * Used to dump a table in case a significantly destructive behavior is to be
+ * done. This should make running a down migration possible if required.
+ *
+ * @param tableName
+ * @returns {Promise<void>}
+ */
+const dumpTable = async tableName => {
+  try {
+    const dumpFilePath = path.resolve(
+      `${__dirname}/../dumps/dumpTable_${tableName}_${new Date().getTime()}.sql`
+    );
+
+    await exec(
+      `pg_dump -t '"${tableName}"' ${process.env.PGDATABASE} --inserts > ${dumpFilePath}`
+    );
+  } catch (err) {
```

Comment:
Do we want to catch this? Might be safer to not run the migration if making the dump fails.

Reply:
Good point! Addressed in https://github.com/w3c/aria-at-app/pull/1187/commits/72861c2854fa6867767309b03ae27b8255504628